config/jest/test-setup.js

```diff
@@ -20,40 +19,6 @@ StyleSheetTestUtils.suppressStyleInjection();
 Enzyme.configure({adapter: new EnzymeAdapter()});
 expect.addSnapshotSerializer(enzymeSerializer);
 
-// Copied from jest-enzyme.
```

Comment:
Would this still be a problem? or is it ok if we leave that wrong JSDOM env open in the meantime until we fully migrate to RTL?

Reply:
I'll see which version of JSDOM the tests are actually running with.  If it's an old version I can add a resolution to our `package.json`.

Reply:
I added a test to verify that we're using v16 of JSDOM in our tests.  No changes appear to be necessary in terms of how we're consuming `jest-enzyme`.